connect/mirror/src/main/java/org/apache/kafka/connect/mirror/MirrorConnectorConfig.java

```diff
@@ -320,6 +321,10 @@ String entityLabel() {
                     in(Utils.enumOptions(SecurityProtocol.class)),
                     ConfigDef.Importance.MEDIUM,
                     CommonClientConfigs.SECURITY_PROTOCOL_DOC)
+            .define(CONFIG_PROVIDERS_CONFIG,
+                    ConfigDef.Type.LIST,
+                    Collections.emptyList(),
+                    ConfigDef.Importance.LOW, CONFIG_PROVIDERS_DOC)
```

Comment:
Nit: Put the DOC on a new line like the other configs. Same in other files below.